packages/experimental/Checkbox/src/Checkbox.types.ts

```diff
@@ -38,6 +39,13 @@ export interface CheckboxProps extends Omit<IViewProps, 'onPress'> {
    */
   checked?: boolean;
 
+  /**
+   * Allows you to set the checkbox to have circular styling.
+   *
+   * @platform Android, iOS, windows, win32
```

Comment:
I don't think you need the extra space? Unless you saw something I didn't

Reply:
Do you mean the extra line? I usually add them in as personal preference, but I can take it out.

Reply:
Yes the extra line. I think it's fine to leave it then, as long as it shows up in intellisense.

Reply:
Yeah it shows just fine in intellisense. I just find the extra line break to help me with parsing